libs/pcre/Makefile

```diff
@@ -57,6 +61,7 @@ CONFIGURE_ARGS += \
 	--enable-utf8 \
 	--enable-unicode-properties \
 	--enable-pcre16 \
+	$(if $(CONFIG_PCRE_JIT_ENABLED),--enable-jit,--disable-jit) \
```

Comment:
Please add `CONFIG_PCRE_JIT_ENABLED` to `PKG_CONFIG_DEPENDS`

Reply:
Will do, thank you!

Reply:
Done!